src/uidl-definitions/types/index.ts

```diff
@@ -120,6 +128,7 @@ export interface WebManifest {
   orientation?: string
   scope?: string
   theme_color?: string
+  screenshots?: Array<{ src: string; sizes: string; type: string }>
```

Comment:
What does this represent?

Reply:
It just holds the screenshots of the App but now i don't think it is not used much after [referring](https://www.w3.org/TR/appmanifest/#screenshots-member) to few examples, will remove that :D